src/components/Radio/Radio.style.ts

```diff
@@ -47,53 +48,56 @@ export const customRadioWrapperStyles = (
   ${focused && !disabled && focusedRadio};
 `;
 
+const determineBoxShadow = ({
+  checked,
+  disabled,
+  filled,
+}: Pick<Props, 'checked' | 'disabled' | 'filled'>) => (theme: Theme) => {
+  if (disabled && !checked) {
+    return `inset 0px 0px 0px ${rem(filled ? '12px' : '2px')} ${theme.utils.getColor(
```

Comment:
you can assign `inset 0px 0px 0px ${rem(...)}` in a variable since it used multiple times.

example
```
const getShadowDimensions = (customProp: number) => `inset 0px 0px 0px ${rem(customProp)}`;

.... 
`${getShadowDimensions(12)} ...`
```